tests/integration/SdkTest_test.cpp

```diff
@@ -644,6 +644,13 @@ void SdkTest::onRequestFinish(MegaApi *api, MegaRequest *request, MegaError *e)
     case MegaRequest::TYPE_FETCH_GOOGLE_ADS:
         mApi[apiIndex].mStringMap.reset(mApi[apiIndex].lastError == API_OK ? request->getMegaStringMap()->copy() : nullptr);
             break;
+
+    case MegaRequest::TYPE_GET_ATTR_NODE:
+        if (mApi[apiIndex].lastError == API_OK)
+        {
+            mMegaFavNodeList = request->getMegaNodeList()->copy();
```

Comment:
You are accumulating a memory leak here, every time you make a copy, the previous allocated memory is leaked.

Reply:
Fixed in 8f5d15a